spring-pulsar-test/src/main/java/org/springframework/pulsar/test/support/PulsarConsumerTestUtil.java

```diff
@@ -53,6 +55,30 @@ public class PulsarConsumerTestUtil<T> implements TopicSpec<T>, SchemaSpec<T>, C
 
 	private List<String> topics;
 
+	private boolean untilMethodAlreadyCalled = false;
+
+	public static <T> TopicSpec<T> consumeMessages() {
+		if (!PulsarTestContainerSupport.isContainerStarted()) {
```

Comment:
We have the logic flipped here (need to remove the `!`).

Reply:
gotchu